### DISCUSSION TITLE
Expiration date

### DISCUSSION THREAD

**Discussion by l4nj1n9** [jonaswinkler/paperless-ng#1473] [Feature Requests]
Sometimes, you don't need to keep some documents all your like. Sometimes, you want to keep them only for one, two, three years. In this case, it should be cool to be able to set an expiration date : when the date is over, paperless-ng should delete the file and its associated datas.

**Comment by capi** [comment]
Maybe instead of deleting, assign a special tag on expiration? (Similar to the Inbox Tag.)
This way no destructive and potentially dangerous operation would have to be performed, but one could easily filter for expired documents and manually delete them.

**Comment by mpfl** [comment]
Automated deletion of items is poor records management practice. That being said, assigning a tag upon an expiry date would be useful.

In my country, you only need to maintain tax records for seven years. Having just bulk ingested ~800 documents into paperless, I have tax records going back 20 years!
